frontend/src/pages/pipelines/global/PipelineCoreNoProjects.tsx

```diff
@@ -9,17 +15,20 @@ const PipelineCoreNoProjects: React.FC = () => {
 
   return (
     <EmptyState>
-      <EmptyStateIcon icon={WrenchIcon} />
-      <Title headingLevel="h4" size="lg">
-        No data science projects
-      </Title>
+      <EmptyStateHeader
+        titleText="No data science projects"
+        icon={<EmptyStateIcon icon={WrenchIcon} />}
+        headingLevel="h4"
+      />
       <EmptyStateBody>
         To create a pipeline server and import a pipeline, first create a data science project.
       </EmptyStateBody>
-      <NewProjectButton
-        closeOnCreate
-        onProjectCreated={(projectName) => navigate(`/pipelines/${projectName}`)}
-      />
+      <EmptyStateFooter>
```

Comment:
No need to use `EmptyStateFooter` here, it will create an extra top margin for the button and make the space too large.

<img width="934" alt="Screenshot 2023-11-03 at 10 53 33 AM" src="https://github.com/opendatahub-io/odh-dashboard/assets/37624318/f95c2940-d4c5-4156-bedd-98ce7cb24cca">

Reply:
This spacing issue seems to have been resolved with ODH dashboard's latest updates to main.

<img width="715" alt="Screenshot 2023-11-16 at 4 49 20 PM" src="https://github.com/opendatahub-io/odh-dashboard/assets/32821331/625cdb8f-bfec-45ae-b641-fcd601a47dc8">

But just as a friendly FYI, based on our [empty state PF docs](https://www.patternfly.org/components/empty-state#basic), the PF recommendation is to keep all empty state actions in EmptyStateFooter